shoppingcart/shopping_cart.py

```diff
@@ -8,6 +8,6 @@
 
 if __name__ == '__main__':
     CloudState()\
-        .port('8080')\
+        .port('8090')\
```

Comment:
Did you choose 80**90** by intention?

Reply:
A random door chosen so as not to conflict with my local environment. It can be anything because it is an example and the proyx could be configured to look for the user role anywhere, just to make it clear that the user can choose the port that suits him.

Reply:
Feel free to suggest changes